tests/Microsoft.Identity.Test.Unit.net45/Resources/MultiTenantTokenCache.json

```diff
@@ -0,0 +1,143 @@
+      "client_info": "eyJ1aWQiOiI2ZWVkYTNhMS1jM2I5LTRlOTItYTk0ZC05NjVhNTBjMDZkZTciLCJ1dGlkIjoiNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3In0",
+      "client_id": "1d18b3b0-251b-4714-a02a-9956cec86c2d",
+      "secret": "secret",
+      "credential_type": "AccessToken",
+      "realm": "49f548d0-12b7-4169-a390-bb5304d24462",
+      "target": "openid profile User.Read email",
+      "user_assertion_hash": null,
+      "cached_at": "1556523684",
+      "expires_on": "1556527283",
+      "extended_expires_on": "1556527283",
+      "ext_expires_on": "1556527283"
+    }
+  },
+  "RefreshToken": {
+    "6eeda3a1-c3b9-4e92-a94d-965a50c06de7.72f988bf-86f1-41af-91ab-2d7cd011db47-login.windows.net-refreshtoken-1d18b3b0-251b-4714-a02a-9956cec86c2d--": {
+      "home_account_id": "6eeda3a1-c3b9-4e92-a94d-965a50c06de7.72f988bf-86f1-41af-91ab-2d7cd011db47",
+      "environment": "login.windows.net",
+      "client_info": "eyJ1aWQiOiI2ZWVkYTNhMS1jM2I5LTRlOTItYTk0ZC05NjVhNTBjMDZkZTciLCJ1dGlkIjoiNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3In0",
+      "client_id": "1d18b3b0-251b-4714-a02a-9956cec86c2d",
+      "secret": "secret",
+      "credential_type": "RefreshToken",
+      "family_id": null
+    },
+    "ae821e4d-f408-451a-af82-882691148603.49f548d0-12b7-4169-a390-bb5304d24462-login.windows.net-refreshtoken-1d18b3b0-251b-4714-a02a-9956cec86c2d--": {
+      "home_account_id": "ae821e4d-f408-451a-af82-882691148603.49f548d0-12b7-4169-a390-bb5304d24462",
+      "environment": "login.windows.net",
+      "client_info": "eyJ1aWQiOiJhZTgyMWU0ZC1mNDA4LTQ1MWEtYWY4Mi04ODI2OTExNDg2MDMiLCJ1dGlkIjoiNDlmNTQ4ZDAtMTJiNy00MTY5LWEzOTAtYmI1MzA0ZDI0NDYyIn0",
+      "client_id": "1d18b3b0-251b-4714-a02a-9956cec86c2d",
+      "secret": "secret",
+      "credential_type": "RefreshToken",
+      "family_id": null
+    }
+  },
+  "IdToken": {
+    "6eeda3a1-c3b9-4e92-a94d-965a50c06de7.72f988bf-86f1-41af-91ab-2d7cd011db47-login.windows.net-idtoken-1d18b3b0-251b-4714-a02a-9956cec86c2d-72f988bf-86f1-41af-91ab-2d7cd011db47-": {
+      "home_account_id": "6eeda3a1-c3b9-4e92-a94d-965a50c06de7.72f988bf-86f1-41af-91ab-2d7cd011db47",
+      "environment": "login.windows.net",
+      "client_info": "eyJ1aWQiOiI2ZWVkYTNhMS1jM2I5LTRlOTItYTk0ZC05NjVhNTBjMDZkZTciLCJ1dGlkIjoiNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3In0",
+      "client_id": "1d18b3b0-251b-4714-a02a-9956cec86c2d",
+      "secret": "foo.eyJhdWQiOiIxZDE4YjNiMC0yNTFiLTQ3MTQtYTAyYS05OTU2Y2VjODZjMmQiLCJpc3MiOiJodHRwczovL2xvZ2luLm1pY3Jvc29mdG9ubGluZS5jb20vNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3L3YyLjAiLCJpYXQiOjE1NTY1MjMxODIsIm5iZiI6MTU1NjUyMzE4MiwiZXhwIjoxNTU2NTI3MDgyLCJuYW1lIjoiVXNlciBJbiB0ZW5hbnQgMSIsIm9pZCI6IjZlZWRhM2ExLWMzYjktNGU5Mi1hOTRkLTk2NWE1MGMwNmRlNyIsInByZWZlcnJlZF91c2VybmFtZSI6InVzZXJAZG9tYWluLmNvbSIsInN1YiI6InJiXzFpRE1KdGNyLWZxTFVsV2RieGljZGxJbGxMbnA3OEpfN2lMSF8tR3ciLCJ0aWQiOiI3MmY5ODhiZi04NmYxLTQxYWYtOTFhYi0yZDdjZDAxMWRiNDciLCJ1dGkiOiJPdnktZmJWZ0UwZTFQVGxIU2UwTkFBIiwidmVyIjoiMi4wIn0=.bar",
```

Comment:
We use the 2nd part of the secret here when creating an AuthneticationResult from an AT and IDT. THis is base64 encoded so I encoded / modified / decoded to make sure there is no PII.